### ISSUE TITLE
ValueError: need more than 2 values to unpack

### ISSUE THREAD

**Issue by jayvdb** [smarkets/flake8-strict#1]
While parsing https://github.com/wikimedia/pywikibot-core/blob/master/pywikibot/data/api.py 

```
$ flake8 --version
2.2.5 (pep8: 1.5.7, flake8_copyright: 0.1, flake8-string-format: 0.2.2.dev0, flake8-print: 2.0.0, pyflakes: 0.8.2a0, mccabe: 0.2.1, flake8-todo: 0.3, import-order: 0.5.1, flake8-blind-except: 0.1.0, flake8_coding: 1.1.0, flake8-quotes: 0.0.1, flake8-future-import: 0.3.2, flake8_strict: 0.1.1, pep257: 0.2.1, flake8-debugger: 1.2, naming: 0.2.2) CPython 2.7.10 on Linux
$ flake8 pywikibot/data/api.py
Traceback (most recent call last):
  File "/usr/bin/flake8", line 9, in <module>
    load_entry_point('flake8==2.4.1', 'console_scripts', 'flake8')()
  File "/usr/lib/python2.7/site-packages/flake8/main.py", line 32, in main
    report = flake8_style.check_files()
  File "/usr/lib/python2.7/site-packages/pep8.py", line 1672, in check_files
    runner(path)
  File "/usr/lib/python2.7/site-packages/flake8/engine.py", line 96, in input_file
    return fchecker.check_all(expected=expected, line_offset=line_offset)
  File "/usr/lib/python2.7/site-packages/pep8.py", line 1412, in check_all
    self.check_ast()
  File "/usr/lib/python2.7/site-packages/pep8.py", line 1359, in check_ast
    for lineno, offset, text, check in checker.run():
  File "/usr/lib/python2.7/site-packages/flake8_strict.py", line 40, in run
    for line, column, error_code in errors:
  File "/usr/lib/python2.7/site-packages/flake8_strict.py", line 77, in _process_parameters
    open_parenthesis, args_list, close_parenthesis = parameters.children
ValueError: need more than 2 values to unpack
```

**Comment by jstasiak** [comment]
Hey @jayvdb, thank you for the feedback and apologies for the delay. This is fixed in master now, we'll do a release some time soon but I can't promise any date.

**Comment by jstasiak** [comment]
@jayvdb This is now released on PyPI as part of 0.1.2
